app/Models/User.php

```diff
@@ -17,11 +17,7 @@ class User extends Authenticatable
      *
      * @var array<int, string>
      */
-    protected $fillable = [
-        'name',
-        'email',
-        'password',
-    ];
+    protected $guarded = [];
```

Comment:
Ahhah why not 😁 although it's a bit risky on User 🙂

Reply:
As far as there is no registration form available and no mass assignement possible, it is okay to leave it this way